chingu-coders/Voyage2-Turtles-02
# add pomodoro settings number inputs

## Purpose
* Add `input type=number` to pomdoro settings
* address issue #110 

## Approach/Fixes
* Feature addition

## Testing
* Checkout this PR and fire up `gulp`
* goto pomodoro settings and click on the number next to slider
* enter a number and confirm that pomodoro period is updated
* test all settings to make sure nothing is broken :) thanks.
